Add note on allowing '_times' for repeated timestamp fields (#539)
googleapis/aip: aip/0142.md

(which uses UNIX timestamps under the hood and hold nanosecond precision).

These fields **should** have names ending in `_time`, such as `create_time` or

> `update_time`. For repeated timestamp fields, names ending in `_times` are also
> allowable.

Reviewer: ```suggestion
`update_time`. For repeated fields, the names **should** end in `_times` instead.
```
